packages/shift-next/src/components/with-checkout.js

```diff
@@ -6,6 +6,10 @@ import Router from 'next/router'
 // Libs
 import { suffixWithStoreName } from '../lib/suffix-with-store-name'
 
+// Next config
+import getConfig from 'next/config'
+const { publicRuntimeConfig: { PAYPAL_CLIENT_ID } } = getConfig()
```

Comment:
```suggestion
const { publicRuntimeConfig: { PAYPAL_CLIENT_ID: payPalClientID } } = getConfig()
```

Then you might be able to delete `const payPalClientID = PAYPAL_CLIENT_ID` on line 122

Reply:
also I'd change `payPalClientID` to `paypalClientID`

Reply:
oooh very nice, will test now